docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -1575,3 +1575,108 @@ module.exports = () => ({
   },
 });
 ```
+
+### Role-Based Access Control <BronzeBadge link="https://strapi.io/pricing"/> <SilverBadge link="https://strapi.io/pricing"/> <GoldBadge link="https://strapi.io/pricing" withLinkIcon/>
+
+::: warning 🚧 This API is considered unstable for now.
+<br>
+:::
+
+Role-Based Access Control (RBAC) is an approach to restricting access to some users. In a Strapi application, users of the admin panel are administrators. Their roles and permissions are [configured in the admin panel](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md). In addition to the 3 default roles (Author, Editor, and Super Admin), it's possible to create custom conditions to match specific needs of roles and permissions management. This requires an Enterprise Edition with at minimum a Bronze licence plan.
+
+#### Defining a new condition
+
+Conditions are defined as an array of objects. Each condition object can have 4 possible properties:
+
+- `displayName` (string): the condition name as shown in the admin panel
+- `name` (string): the condition name, kebab-cased
+- `plugin` (string, optional): if the condition is created by a plugin name, should be the plugin's name, kebab-cased (e.g `content-manager`)
+- `handler` (function): a function to handle the user and possibly return a `condition object` (see [using the handler function](#using-the-handler-function))
```

Comment:
The handler can also be a condition object (same format as the object that can be returned from a handler function)

Reply:
Hmm, OK, looks like the Handler part of this doc needs to be expanded, then. Would you mind giving me an example or a link to the strapi codebase, please?

Reply:
Thank you for your explanations during our meeting :-) I [updated the doc](https://github.com/strapi/documentation/commit/9c18431c869d7fac5f3dd8a0aba6a2dd18744319) to reflect on the fact that the `handler` can either be an object or a function